cli/README.md

```diff
@@ -34,3 +34,19 @@ help:
 ```bash
 popper --help
 ```
+
+## Bash-completion
+To enable auto-completion support, you will need to activate the popper.sh script
+inside the `cli/extras` folder. This can be done by copying this script to 
+`/etc/bash_completion.d/` or `/usr/local/etc/bash_completion.d/`.To do this, 
+go to the extras folder and copy and paste the following commands in your terminal :- 
```

Comment:
bash-completion is not available by default on bash, so we should maybe explicitly mention that it has to be installed/enabled separately. [this](https://github.com/bobthecow/git-flow-completion/wiki/Install-Bash-git-completion) is a good guide that we could link to, since it has instructions for many distinct OSs.

Reply:
Yes, I will a proper guide to this.